### PR TITLE
Feature Selector - Information Gain

### PR DESCRIPTION
**Pull request by IuliaUngur**

### CODE REVIEW

**Comment by IuliaUngur** [comment]
Issue #31

**Comment by IuliaUngur** [comment]
Issue #26

**Comment by IuliaUngur** [comment]
Issue #28

File: app/services/ap/algorithm/fselector_filter.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        @ranks
+      end
+
+      private
+
+      def initialize_selector
+        @r = FSelector::IG.new
+
+        name2type = {
+            light: :string,
```
Comment (iuliasincai) [inline app/services/ap/algorithm/fselector_filter.rb]: Just some extra spaces here

**Pull request merged**